repository/src/main/resources/db/changelog/db.changelog-rdm-4426.xml

```diff
@@ -0,0 +1,32 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="rdm-4426" author="rafal.kalita@hmcts.net">
+        <addColumn tableName="search_input_case_field">
+            <column name="case_field_element_path" type="varchar(300)"/>
+        </addColumn>
+        <addColumn tableName="search_result_case_field">
+            <column name="case_field_element_path" type="varchar(300)"/>
+        </addColumn>
+        <addColumn tableName="workbasket_case_field">
+            <column name="case_field_element_path" type="varchar(300)"/>
+        </addColumn>
+        <addColumn tableName="workbasket_input_case_field">
+            <column name="case_field_element_path" type="varchar(300)"/>
+        </addColumn>
+
+        <dropUniqueConstraint constraintName="unique_search_input_case_field_case_type_id_case_field_id"
```

Comment:
Dropping the constraints and moved validation to the code, so it will be in one place. The alternative was to create the unique constraints with three elements, of which one can be null.